### ISSUE TITLE
Shuttle issues

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#4324]
This is for tracking problem as they exist on master:

- [x] OccluderTree doesn't update properly. Fixed in https://github.com/space-wizards/RobustToolbox/pull/1878
- [x] Item lerping looks funny. I think it's going from a map position instead.
- [x] Anchoring uses worldposition instead of EntityCoordinates maybe?

**Comment by hubismal** [comment]
Not sure how strictly of a "shuttle" issue this is, but at least in sandbox, construction and subfloor construction (APCs and cables) don't seem to work on "shuttles" I build outside the main ship (by free-placing tiles a distance out from the station), but placing them right outside the shuttle works and they build a grid that seems to be independent of the shuttle. It's possible the game still thinks you're trying to build onto the main station when you try and place things down on the shuttle.
tl;dr placing APCs and cables are effed for some reason on landmasses that aren't the main station

**Comment by metalgearsloth** [comment]
Definitely related, I'll suss it out.

**Comment by hubismal** [comment]
An additional cosmetic issue that may be related is the pickup animation for items (where it glides from its position on the floor to your character) seems to originate from the main station somewhere instead of its shuttle position when you pick something up in a shuttle (it flies across the screen to your character).

**Comment by hubismal** [comment]
To expand on this, it looks like placing cables and so on also doesn't work if the ship has been moved.

**Comment by hubismal** [comment]
![image](https://user-images.githubusercontent.com/47284081/128411998-e7e2f44b-a86e-4f56-9c8c-6b4268d2344e.png)
Unsure how much this counts, but the colliders for ship bodies seems to be a little off. in some crevices and nooks, there are invisible walls.

**Comment by hubismal** [comment]
This could also be unrelated, but the atmos doesn't seem to work in shuttles as well

**Comment by Zumorica** [comment]
Atmos simulation should work, gas overlays are broken on offset grids currently.

**Comment by hubismal** [comment]
![image](https://user-images.githubusercontent.com/47284081/128416718-d9935b8c-626b-4d58-87ba-b3791adf5b86.png)
Once a ship moves, placing down linking tiles seems to be broken for certain things like low walls and tables

**Comment by metalgearsloth** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/47284081/128411998-e7e2f44b-a86e-4f56-9c8c-6b4268d2344e.png)
> Unsure how much this counts, but the colliders for ship bodies seems to be a little off. in some crevices and nooks, there are invisible walls.

Known, needs coding engine-side which will be an effort and a half

**Comment by metalgearsloth** [comment]
Will make a new issue that will include rotation stuff too.
